surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformEnginesIT.java

```diff
@@ -395,28 +396,46 @@ public void testJupiterEngineWithDisplayNames()
-                .assertContainsText( "73$72 ✔" );
+                .assertContainsText( "73$71 ✔" )
+                .assertContainsText( "73$72 ✔" )
+                .assertContainsText( "73$73 ✔" )
+                .assertContainsText( "73$74 ✔" );
 
 
         validator.getSurefireReportsFile( "TEST-junitplatformenginejupiter.DisplayNameTest.xml", UTF_8 )
                 .assertContainsText( "testcase name=\"73$71 ✔\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
                 .assertContainsText( "testcase name=\"73$72 ✔\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
+                .assertContainsText( "testcase name=\"73$73 ✔ [1] a\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
+                .assertContainsText( "testcase name=\"73$73 ✔ [2] b\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
+                .assertContainsText( "testcase name=\"73$73 ✔ [3] c\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
+                .assertContainsText( "testcase name=\"73$74 ✔ [1] a\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
+                .assertContainsText( "testcase name=\"73$74 ✔ [2] b\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
+                .assertContainsText( "testcase name=\"73$74 ✔ [3] c\" classname=\"&lt;&lt; ✨ &gt;&gt;\"" )
                 .assertContainsText( XML_TESTSUITE_FRAGMENT );
 
 
         validator.getSurefireReportsFile( "TEST-junitplatformenginejupiter.BasicJupiterTest.xml", UTF_8 )
                 .assertContainsText( "<testcase name=\"test(TestInfo)\" "
                         + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
-                .assertContainsText( "<testcase name=\"0 + 1 = 1\" "
+                .assertContainsText( "<testcase name=\"add(int, int, int)[1] 0 + 1 = 1\" "
+                        + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
+                .assertContainsText( "<testcase name=\"add(int, int, int)[2] 1 + 2 = 3\" "
+                        + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
+                .assertContainsText( "<testcase name=\"add(int, int, int)[3] 49 + 51 = 100\" "
+                        + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
+                .assertContainsText( "<testcase name=\"add(int, int, int)[4] 1 + 100 = 101\" "
+                        + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
+                .assertContainsText( "<testcase name=\"square(int, int)[1] 1, 1\" "
+                        + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
+                .assertContainsText( "<testcase name=\"square(int, int)[2] 2, 4\" "
+                        + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
+                .assertContainsText( "<testcase name=\"square(int, int)[3] 3, 9\" "
                         + "classname=\"junitplatformenginejupiter.BasicJupiterTest\"" )
-                .assertContainsText( "<testcase name=\"1 + 2 = 3\" "
```

Comment:
Where is the old method? I cannot see it here in this PR.
I would like to keep it and add new ones because then we have more alternatives. Would it be possible?
Pls check our documentation because sometimes we use the reference pointing to our IT tests.

Reply:
I will rewrite the PR, so that it does not change existing ITs. Thanks for your feedback.

Reply:
Rewrote it, so that only a new dedicated test project and IT for it is added.

Test project:    surefire-1914-xml-reporting-parameterizedtest
IT:    Surefire1914XmlReportingParameterizedTestIT.java

Please have a look again.

For reference: The relevant pre-existing IT with a divergent expectation is JUnitPlatformEnginesIT#testJupiterEngineWithDisplayNames